### PR TITLE
Rewrite the reputation commands in Zscaler to return in a CommandResults obj

### PR DESCRIPTION
**Pull request by ilappe**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-18256

## Description
in the current implementation - Zscaler return the `url` and `ip` indicators in one entry 
with an list of debot_score for each url passed in the CSV `url` arg

this cause to cache all of those results in each indicator cache entry
and if an indicator will be extracted - all of those indicators will also return 
this are wrong and cause some issues

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
       - Due to the structure of the CommandResults obj we need to put some context data in a new key
       - for example `Zscaler.IP.ipClassifications` instead of `IP.ipClassifications`

### CODE REVIEW

File: Packs/Zscaler/ReleaseNotes/1_3_5.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Zscaler Internet Access
+- Fixed an issue where the ***ip*** and ***url*** reputation commands return the data in wrong format.
```
Comment (tkatzir) [inline Packs/Zscaler/ReleaseNotes/1_3_5.md:4]:
```suggestion
- Fixed an issue where the ***ip*** and ***url*** reputation commands returned the data in wrong format.
```

File: Packs/Zscaler/Integrations/Zscaler/Zscaler.py [hunk 2]
```diff
@@ -416,123 +419,116 @@ def url_lookup(args):
...
         res_url = data.get('url')
         for url in urls_list:
             # since zscaler expects to recieve a URL without the protocol, we omit it in `lookup_request`
             # in the response, the URL is returned as it was sent, so we add back the protocol by replacing
             # the URL retruned with the one we got as an argument
             if 'http://' + res_url in url or 'https://' + res_url in url:
                 data['url'] = url
+
         ioc_context = {'Address': data['url'], 'Data': data['url']}
         score = 1
```
Comment (tkatzir) [inline Packs/Zscaler/Integrations/Zscaler/Zscaler.py:436]: Can all the score values (0, 1, 2, 3) be in an ENUM? And be given meaningful names?

**Review by tkatzir — COMMENTED** [review]
Looks good!
Some comments.

**Review by tkatzir — APPROVED** [review]

**Comment by ilappe** [comment]
need force merge due to [backwards compatibility break](https://app.circleci.com/pipelines/github/demisto/content/210429/workflows/6b4046d6-076b-43c0-a9f3-a6591b94c2c8/jobs/604249?invite=true#step-108-94)

**Pull request merged**